.vscode/settings.json

```diff
@@ -4,31 +4,43 @@
     "storybook-static": true
   },
   "files.exclude": {
-    "**/storybook-static": true,
+    // Folders we don't need ever
```

Comment:
Comments are not valid json 🤪

Reply:
whoops, I'm working strictly with jsonc files in my private repo's 🙈